kingpin/actors/base.py

```diff
@@ -308,6 +308,15 @@ def _fill_in_contexts(self, context={}, strict=True):
         # Finally, convert the string back into a dict and store it.
         self._options = json.loads(new_options_string)
 
+    def get_orgchart(self, parent=''):
```

Comment:
I think an actor could have `self._exec_data` dictionary with arbitrary data stored in there. Each actor would choose what to store, with some values being expected such as `_exec_data['runtime']`, success status, # api calls, or # of retry fails etc.etc.